dashboard/test/factories/pd_factories.rb

```diff
@@ -474,7 +474,7 @@
     code {SecureRandom.hex(10)}
 
     trait :from_user do
-      user
+      user {create :teacher}
```

Comment:
This fixes a pre-existing bug, the default `user` is a student, that then fails the enrollment validation as it requires an email.